contributing-guides/style-guide.md

```diff
@@ -104,6 +104,10 @@ Example:
 ```
 - Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
 
+## Description syntax
+
+- Use backticks for quoting in descriptions as they highlighted in GitHub.
```

Comment:
I don't understand this change. What does it do? (Can this be used to quote in our clients?)

Reply:
The purpose of this change is to stick one type of quotes in our code descriptions.

Reply:
Note that we already using backticks thanks to our standardization PRs.

Reply:
Backticks are code blocks though? I'm confused.

Reply:
Yes, they are like inlined code blocks.

Reply:
Yes, so it makes absolutely no sense at all to use backticks instead of quotes. Sure, they might be highlighted on github, but they make no sense at all in other contexts, not to mention being semantically incorrect. If quotes are required in an example description, then quotes should be used: not backticks.

Backticks in example descriptions should be limited to actual code only.

Reply:
I've required just double quotes for everything.